common-libs/common/package.json

```diff
@@ -10,7 +10,7 @@
     "tsc": "tsc -p tsconfig-build.json",
     "lint": "eslint src test",
     "test": "npm run test:unit && npm run test:integration",
-    "test:unit": "cross-env NODE_ENV=test mocha -r ts-node/register test/unit/index.test.ts -p ./tsconfig.json --timeout 55000",
+    "test:unit": "cross-env NODE_ENV=test mocha -r ts-node/register test/unit/index.test.ts -p ./tsconfig.json --timeout 55000 --parallel false",
```

Comment:
@anton-iskryzhytskyi should we return parallel run for tests ?

Reply:
is `-p` stands for parallel? it is better do delete `-p` here I suppose